gateway-js/src/datasources/LocalGraphQLDataSource.ts

```diff
@@ -11,8 +11,7 @@ import { GraphQLDataSource, GraphQLDataSourceProcessOptions } from './types';
 
 export class LocalGraphQLDataSource<
   TContext extends Record<string, any> = Record<string, any>,
```

Comment:
Seems like the indentation is inconsistent with this case vs. one I pointed out previously. Maybe the one I pointed out above is a bug?